R/cache.R

```diff
@@ -270,11 +270,14 @@ build_cached_function <- function(cached_function) {
 execute <- function(fcn_call) {
   # Grab the new/old keys
   keys <- fcn_call$call[[fcn_call$key]]
-  if (fcn_call$force) { uncached_keys <- keys }
-  else {
+  if (fcn_call$force) 
+  { 
+    uncached_keys <- keys 
+  } else {
     uncached_keys <- get_new_key(fcn_call$con, fcn_call$table, keys, fcn_call$output_key)
   }
   cached_keys <- setdiff(keys, uncached_keys)
+  stopifnot(remove_old_key(fcn_call$con, fcn_call$table, uncached_keys, fcn_call$output_key))
```

Comment:
Because of `if (!DBI::dbExistsTable(dbconn, tbl_name)) return(FALSE)`, this will prevent from a new caching table ever getting created